hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+This release simplifies the logic of the ``print_blob`` setting by removing the option to set it to ``PrintSettings.INFER``.
+As a result the ``print_blob`` setting now takes a single boolean value, and the use of ``PrintSettings`` is deprecated.
```

Comment:
It would be good to have a cross-reference so users can click-through to the relevant docs:

    :attr:`~hypothesis.settings.print_blob`